### PR TITLE
Fix favorite button on swap list on Android

### PR DESCRIPTION
**Pull request by tchayen**
Fixes RNBW-2575

## What changed (plus any additional context for devs)

We have separate code for this button on iOS and on Android. On Android tap handler was passed incorrectly and in result it was never called.

## Dev checklist for QA: what to test

1. Tap on swap button.
2. Tap on `Choose Token`.
3. Try to add/remove a favorite on the list.

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?

### CODE REVIEW

**Comment by tchayen** [comment]
It's not highlighted in the changeset, but very important piece of code here is there:
https://github.com/rainbow-me/rainbow/blob/develop/src/components/coin-row/ExchangeCoinRow.js#L80

Because it's `onPress` handler, for Android nothing will be passed here to the function (unline on iOS where it's a callback).  `onNewEmoji` param is used in one line: `ios && onNewEmoji();` so it won't cause exception on Android.

So tl;dr this code seems safe to me, but keep that in mind when reviewing and please have a look there.

**Review by osdnk — APPROVED** [review]

**Review by skylarbarrera — APPROVED** [review]

**Review by brunobar79 — APPROVED** [review]
LGTM!

**Review by ibrahimtaveras00 — APPROVED** [review]
Looks good on Android now, and also spot checked iOS to be safe 👍🏽
